sms_service.py

```diff
@@ -2,24 +2,18 @@
 from email.mime.image import MIMEImage
 from email.mime.multipart import MIMEMultipart
 import smtplib
-import sys
 import os
- 
-CARRIERS = {
-    "att": "@mms.att.net",
-    "tmobile": "@tmomail.net",
-    "verizon": "@vtex.com",
-    "sprint": "@page.nextel.com"
-}
- 
-EMAIL = ""
-PASSWORD = ""
-
-RECIPIENTS = []
- 
+import security as vault
+
+EMAIL = vault.get_value("credentials", "sms_auth", "username")
+PASSWORD = vault.get_value("credentials", "sms_auth", "password")
+CARRIER_ADDR = vault.get_value("carriers", "tmobile", "address")
```

Comment:
Maybe we shouldn't have these be global variables :D

Reply:
Great point, this is a little unsafe. Removed! 😃